docs/using-avail/getting-started/explorer.md

```diff
@@ -645,7 +629,7 @@ one you wish to get availability guarantees for.
 
 ```bash
 
-curl -s -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","method":"get_blockConfidence","params": {"number": block_number_here}, "id": 1}' 'https://polygon-da-light.matic.today/v1/json-rpc'
+curl -s -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","method":"get_blockConfidence","params": {"number": block_number_here}, "id": 1}' 'https://testnet.avail.tools/v1/json-rpc'
```

Comment:
unrelated to docs: should we separate LC endpoints from node endpoints? 
cc: @jakubcech